.travis.yml

```diff
@@ -6,3 +6,6 @@ rvm:
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+addons:
+  code_climate:
+    repo_token:
```

Comment:
@gjtorikian I’m not authorized to see the token. Could you please look up and paste it here?